planet-ecommerce/src/pages/productListing/ProductListing.jsx

```diff
@@ -0,0 +1,156 @@
+import "./productlisting.css"
+import { CardVertical } from "../../components/card/CardVertical";
+import { Navigation } from "../../components/navigation/Navigation";
+import { useEffect, useState, useContext } from 'react';
+import axios from 'axios';
+import { AsideBar } from "../../components/asidebar/AsideBar";
+import { ProductPageContext } from "../../context/productPageContext";
+import { CartContext } from "../../context/cartContext";
+import { WishlistContext } from "../../context/wishListContext";
```

Comment:
you can import all the components in one file and then import all of them in a single line from that file.
`e.g: import {ComponentA,ComponentB,....} from "../../components/index.js"`